stable/nginx-ingress/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.33.5
+version: 1.33.6
```

As we have added a feature to the chart please bump the minor version.
```suggestion
version: 1.34.0
```

---

@ChiefAlexander I have addressed both of you comments but please have a look at the readme file after the upadte